index.js

```diff
@@ -2,12 +2,7 @@ module.exports = function(content) {
   this.cacheable && this.cacheable();
   this.value = content;
 
-  var array = new Array();
-  for (var i = 0; i != content.length; ++i) {
-    array[i] = content[i];
-  }
-
-  return 'module.exports = new Buffer(' + JSON.stringify(array) + ')';
+  return 'module.exports = new Buffer("' + content.toString('base64') + '", "base64")';
```

Comment:
As per #3 calls to `new Buffer()` are runtime deprecated in Node 10. This could be resolved using `Buffer.from(string, encoding)` instead.